src/main/kotlin/app/revanced/patches/music/layout/permanentshuffle/fingerprints/PermanentShuffleFingerprint.kt

```diff
@@ -0,0 +1,20 @@
+package app.revanced.patches.music.layout.permanentshuffle.fingerprints
+
+import app.revanced.patcher.extensions.or
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import org.jf.dexlib2.AccessFlags
+import org.jf.dexlib2.Opcode
+
+object PermanentShuffleFingerprint : MethodFingerprint(
```

Comment:
Give this fingerprint a better name, for example, "DisableShuffleFingerprint", since you return it early to permanently shuffle.